lego/templates/email/base.html

```diff
@@ -216,27 +216,17 @@
         <table cellspacing="0" cellpadding="0" width="100%" bgcolor="#ffffff" background="http://s3.amazonaws.com/swu-filepicker/4E687TRe69Ld95IDWyEg_bg_top_02.jpg" style="background-color:transparent">
           <tr>
             <td width="100%" height="80" valign="top" style="text-align: center; vertical-align:middle;">
-            <!--[if gte mso 9]>
-            <v:rect xmlns:v="urn:schemas-microsoft-com:vml" fill="true" stroke="false" style="mso-width-percent:1000;height:80px; v-text-anchor:middle;">
-              <v:fill type="tile" src="http://s3.amazonaws.com/swu-filepicker/4E687TRe69Ld95IDWyEg_bg_top_02.jpg" color="#ffffff" />
-              <v:textbox inset="0,0,0,0">
-            <![endif]-->
```

Comment:
This code applies to anyone using outlook. I'm not sure what it does myself, but I'm guessing there is a reason we have it :sweat_smile:. Maybe someone with outlook can test?

Reply:
Nice spot! I though it was only some old comments left in the code.😅

I see no point deleting it now since it might have a function.

Reply:
Could you readd this, so we can merge this? :smile: